README.md

```diff
@@ -63,129 +76,140 @@ master
 Standard IntelliJ dialog for interactive rebase will appear.
 
 ![](docs/rebase.gif)
+</details>
 
-### Push
+<details>
+<summary>Push</summary>
 
 After a rebase, you might want to push the rebased branch to the remote.
-Right-click on the given branch and select `Push (Current) Branch` from context menu.
-Push dialog will appear with proper push button (regular push or force push) depending on which of them is needed.
-Basically, we need force push in case when we rebased our branch, because local and remote branch diverged from each other (they have different commits), so git won't let us do regular push.
+Right-click on the given branch and select `Push (Current) Branch` from a context menu.
+Push dialog will appear with a proper push button (regular push or force push) depending on which of them is needed.
+Basically, we need force push in a case when we rebased our branch because local and remote branch diverged from each other (they have different commits), so git won't let us do regular push.
 
 ![](docs/push.gif)
+</details>
 
-### Pull
+<details>
+<summary>Pull</summary>
 
-If branch is behind (or in sync with) its remote, it can be pulled.
-To do this, right-click on given branch and select `Pull Branch`.
+If a branch is behind (or in sync with) its remote, it can be pulled.
+To do this, right-click on a given branch and select `Pull Branch`.
 This action does `--ff-only` pull only which is enough in this case.
 
 ![](docs/pull.gif)
+</details>
 
-### Reset to remote
+<details>
+<summary>Reset to remote</summary>
 
-If given branch is not in sync with its remote, we can reset it to point to the exact same commit as the remote.
-From right-click context menu select `Checkout and Reset Branch to Remote`.
+If a given branch is not in sync with its remote, we can reset it to point to the exact same commit as the remote.
+From a right-click context menu select `Checkout and Reset Branch to Remote`.
 This action performs `git reset --keep` under the hood. `--keep`, as opposed to `--hard`, keeps your uncommitted changes safe from getting overwritten.
 
 ![](docs/reset.gif)
+</details>
 
-### Fast forward parent
+<details>
+<summary>Fast forward parent</summary>
 
 When you are done with the changes on the selected branch (e.g. PR was approved and all required fixes are applied), you probably want to merge this branch to its parent branch.
 When these branches are in sync (green edge) you can do a fast-forward merge.
-Fast forward merge is nothing else like move tip of branch to which we merge to the same commit as tip of merged branch.
-This is what `Fast Forward Parent Branch To Match Current Branch` context menu action do.
-As you can see, edge between given branch and its parent was changed to grey - it means that the child branch has been merged.
+Fast forward merge is nothing else like move tip of a branch to which we merge to the same commit as a tip of a merged branch.
```

Comment:
Nit: use `fast-forward` (preferred) or `fast forward` consistently

Reply:
`s/nothing else like move/just equivalent to moving`

Reply:
In plugin's user-facing messages it's `Fast forward` so I'd stuck with that.